src/components/erc721/collectibles/collectibles_all.tsx

```diff
@@ -159,7 +161,7 @@ export class CollectiblesAll extends React.Component<Props> {
 
 const allMapStateToProps = (state: StoreState): StateProps => {
     return {
-        collectibles: getOtherUsersCollectibles(state),
+        collectibles: getAllCollectibles(state),
```

Comment:
Posted [a question](https://github.com/0xProject/0x-launch-kit-frontend/issues/521#issuecomment-499187489) in the corresponding issue about the criteria that the collectibles should meet here.

Reply:
It looks like we should revert to the `getOherUsersCollectibles` and append the current user's collectibles that **are on sale**. Check [this comment](https://github.com/0xProject/0x-launch-kit-frontend/issues/521#issuecomment-499198412) and the answer below.

Reply:
Done in this [commit](https://github.com/0xProject/0x-launch-kit-frontend/pull/531/commits/827eb390cc33bff37b3e81ce27192aa819914c4e)